examples/Get-GPPPassword.py

```diff
@@ -149,6 +158,7 @@ def parse_args():
     parser = argparse.ArgumentParser(add_help=True,
                                      description='Group Policy Preferences passwords finder and decryptor')
     parser.add_argument('target', action='store', help='[[domain/]username[:password]@]<targetName or address>')
```

Comment:
```suggestion
    parser.add_argument('target', action='store', help='[[domain/]username[:password]@]<targetName or address> or LOCAL'
                                                       ' (if you want to parse local files)')
```

Reply:
Like `secretsdump`, it would be nice to add the argument LOCAL to the help section.